### PR TITLE
Modules 5896 fix for rubocop

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**

### CODE REVIEW

File: lib/puppet/provider/reboot/windows.rb [hunk 1]
```diff
@@ -148,8 +152,8 @@ def package_installer?
     # 0x00000000 (0)	No pending restart.
     path = 'SOFTWARE\Microsoft\Updates'
     value = reg_value(path, 'UpdateExeVolatile')
-     # this may error if this is not a control flow construct
-    if value and value != 0
+    # this may error if this is not a control flow construct
+    if value && value != 0
```
Comment (Iristyle) [inline lib/puppet/provider/reboot/windows.rb]: I think I changed one of these at one point and ... things exploded.

Comment (Iristyle) [reply lib/puppet/provider/reboot/windows.rb]: Yup, my change made sense to me at the time as part of https://github.com/puppetlabs/puppetlabs-reboot/pull/100

It ended up being reverted in https://github.com/puppetlabs/puppetlabs-reboot/pull/104

We'll have to revisit this...

**Comment by glennsarti** [comment]
@ThoughtCrhyme  Needs a rebase

**Comment by glennsarti** [comment]
Travis CI failures are not due to this PR... but external to this module.

**Pull request merged**